### PR TITLE
Report the type of build environment in use.

### PR DESCRIPTION
**Pull request by sdake**

### CODE REVIEW

File: files/Makefile [hunk 1]
```diff
@@ -55,6 +55,7 @@ REPO_NAME = $(shell basename $(REPO_ROOT))
 TARGET_OUT ?= $(HOME)/istio_out/$(REPO_NAME)
 
 ifeq ($(BUILD_WITH_CONTAINER),1)
+$(info Building with Istio's cononical build container.)
```
Comment (geeknoid) [inline files/Makefile]:
```suggestion
$(info Building with Istio's canonical build container.)
```

Comment (geeknoid) [reply files/Makefile]: How about include the date of the container in this text?

Comment (sdake) [reply files/Makefile]: I thought of that - though it may be too long - will add and see how it looks on nthe terminal - thanks :)

Comment (sdake) [reply files/Makefile]: PTAL.

**Pull request merged**